File: 알고리즘스터디/jisu/programmers/Lv.1/k번째수.js
const solution = (array, commands) => {
  let answer = [];

  commands.map((arr) => {
    let copyArray = array;
    let newArray = array.slice(arr[0] - 1, arr[1]);
    newArray.sort((a, b) => a - b);
    answer.push(newArray[arr[2] - 1]);
  });

  return answer;
};

File: 알고리즘스터디/hyein/programmers/k번째수.java
import java.util.*;
class Solution {
    public int[] solution(int[] array, int[][] commands) {
        int[] answer = new int[commands.length];
        for(int i = 0; i < commands.length; i++) {
            int length = commands[i][1] - commands[i][0];
            int[] subArray = new int[length + 1];
            for(int j = 0; j <= length; j++) {
                subArray[j] = array[commands[i][0] + j - 1];
            }
            
            Arrays.sort(subArray);
            answer[i] = subArray[commands[i][2]-1];
        }
        
        return answer;
    }
}